webapp/src/components/Map.js

```diff
@@ -1,23 +1,30 @@
 import React from 'react';
 import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import L from 'leaflet';
```

Comment:
This import `L` is not used anywhere and building the app shows a warning which will cause the CI to fail
```
Line 3:8:  'L' is defined but never used  no-unused-vars
```